proxies/nodejs/app.ts

```diff
@@ -1,20 +1,53 @@
-import { initialize } from '@devcycle/nodejs-server-sdk'
+import { DVCClient } from '@devcycle/nodejs-server-sdk'
 import Koa from 'koa'
+import Router from 'koa-router'
+import KoaBody  from 'koa-body'
 
-async function start() {
-    await initialize('<TOKEN>').onClientInitialized()
+const data = {
+  clients: {},
+  users: {},
+  commandResults: {}
+}
+
+const handleClient = async (ctx) => {
+  const body = ctx.request.body
+  if (body.clientId === undefined) {
+    ctx.status = 400 
+    ctx.body = "Invalid request: Missing clientId"
+  } else if (!body.sdkKey) {
+    ctx.status = 400 
+    ctx.body = "Invalid request: Missing sdkKey"
+  } else {
+    const client = new DVCClient(ctx.request.body.sdkKey, ctx.request.body.options)
+    data.clients[ctx.request.body.clientId] = client
+    await client.onClientInitialized()
```

Comment:
I dont think you want to wait for this in the client creation endpoint, because we need to be able to do tests that call methods on the client before its finished initializing. Waiting for init would be one of the `:location` requests

Reply:
good point, i'll take it out

Reply:
also i screwed up the branch so i moved this to a new pr https://github.com/DevCycleHQ/test-harness/pull/13